modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/admin/info_panel.jsp

```diff
@@ -67,7 +67,15 @@ if (ListUtil.isEmpty(kbFolders) && ListUtil.isEmpty(kbArticles)) {
 				<div class="autofit-col">
 					<ul class="autofit-padded-no-gutters autofit-row">
 						<li class="autofit-col">
-							<liferay-util:include page="/admin/kb_folder_action.jsp" servletContext="<%= application %>" />
+
```

Comment:
There is some kind of trouble with the actions from the info_panel (maybe the redirection) that previously did not happened. Can you take a look?

https://user-images.githubusercontent.com/47524751/186184995-225cca48-2199-45c1-bb80-4075e85c6733.mp4

Reply:
Fixed.